README.md

```diff
@@ -1,83 +1,26 @@
 Takeaway Challenge
-==================
-```
-                            _________
-              r==           |       |
-           _  //            |  M.A. |   ))))
-          |_)//(''''':      |       |
-            //  \_____:_____.-------D     )))))
-           //   | ===  |   /        \
-       .:'//.   \ \=|   \ /  .:'':./    )))))
-      :' // ':   \ \ ''..'--:'-.. ':
-      '. '' .'    \:.....:--'.-'' .'
-       ':..:'                ':..:'
-
- ```
-
-Instructions
 -------
+<br/>
+The goal was to create a platform that would allow a user to place a takeway order. I decided to break down the project into four classes. The `Dish` and `Menu` classes were created to provide future expansion. I could have set a menu in the initialize method of the `Takeaway` class but I thought this would be cleaner and more future proof. 
```

Comment:
This is a clear, concise README.md., and really helps orientate the reader. Your rationale for the classes makes a lot of sense and it's useful to have this rationale right here at the top. (For lack of time, I did precisely the opposite of what you did and what we were encouraged to do: i.e., divide the project up into multiple classes.) The anticipation of 'future expansion' strikes me as good practice, too.

I also agree that a separate menu is much clearer than including this list in the initialise method of 'Takeaway'.